app/views/state_file/state_file_pages/about_page.html.erb

```diff
@@ -1,18 +1,18 @@
 <% title = t(".header") %>
-<h1 class="h2 strong" style="font-weight: bold"><%= title %></h1>
-<div>
-  <%= t(".subheader_html") %>
-</div><br>
+
+<div class="fyst-home-container">
+  <div class="fyst-home-content">
+    <h1 class="h1 strong" style="font-weight: bold"><%= title %></h1>
+  </div>
+  <%= image_tag 'questions/welcome.svg', class: 'fyst-home-image' %>
+</div>
+
 <div>
-  <%= link_to t(".action1_text"), "" %>
+  <%= t(".subheader_html", login_path: login_options_path(us_state: "az")) %>
```

Comment:
The login pages currently are tied to a state so we should probably route based on the user's state since we wouldn't necessarily have it at this point.

Reply:
We could deduce the state after login and route appropriately.

Reply:
I have a PR at the moment that makes "us" an appropriate option for login - when passed in we then determine which state is required as part of login